changelog/change_filter_map_safety_docs.md

```diff
@@ -0,0 +1 @@
+* [#280](https://github.com/rubocop/rubocop-performance/issues/280): Improved the explanation for the safety of the `MapCompact` autocorrection. ([@Splines][])
```

Comment:
This changelog entry is not necessary for document updates. Can you remove it?

Reply:
sure, I removed it